arch/sim/Kconfig

```diff
@@ -355,6 +355,13 @@ config SIM_FBBPP
 		If you use the X11 display emulation, the selected BPP must match the BPP
 		of your graphics hardware (probably 32 bits).  Default: 8
 
+config SIM_FBCOUNT
```

Comment:
change to SIM_FRAMEBUFFER_COUNT and depends on SIM_FRAMEBUFFER

Reply:
Done.